package.json

```diff
@@ -32,10 +32,18 @@
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@types/react": "^18.0.9",
     "husky": "^8.0.1",
     "lint-staged": "^13.0.3"
   },
   "resolutions": {
     "usehooks-ts@^2.7.2": "patch:usehooks-ts@npm:^2.7.2#./.yarn/patches/usehooks-ts-npm-2.7.2-fceffe0e43.patch"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-tsparticles": "^2.12.2",
+    "tsparticles": "^3.1.0",
+    "tsparticles-engine": "^2.12.0",
+    "tsparticles-slim": "^2.12.0"
```

Comment:
This is root `package.json`, I think you bymistake added all depeendencies here. 

Make sure you first `cd packages/nextjs` and then `yarn add` all the dependencies

Reply:
Whoops! Thanks for spotting that. Corrected !